### DISCUSSION TITLE
Spring element availability

### DISCUSSION THREAD

**Discussion by xyont** [BriefFiniteElementNet/BriefFiniteElement.Net#166] [Ideas]
hi,

spring element is commonly used and basic of element, however it's not yet implemented or i'm missing? the availability of this element in the next or future version can be useful features. Many model such as beam or plate rested on elastic foundation can be solved.

regards,

**Comment by epsi1on** [comment]
The problem with older springs (like `Spring1d` and `ParametricSpring`) was they where not convenient. anyways i think having a new spring element  would be good (as in #167)
thanks for suggestion

**Comment by epsi1on** [comment]
just noticed there is a spring in `CustomElements` library named `ParametricSpring` ([link](https://github.com/BriefFiniteElementNet/BriefFiniteElement.Net/blob/master/Sources/CustomElements/Elements/ParametricSpring.cs)) which is not obsolete and you may use it without any problem

**Reply by xyont** [reply]
many thanks, please provide some simple example to use.

**Reply by epsi1on** [reply]
#169 is to add documentation for ParametricSpring

**Comment by epsi1on** [comment]
inactivity
